terraform/oke/schema.yaml

```diff
@@ -111,47 +100,61 @@ variables:
   #   required: true
 
   ####
-  ##  OCI Observability & Management Services Configuration
+  ##  OCI Observability and Management Services Configuration
   ####
-  
+
+  #  Stack Deployment Options
+  stack_deployment_option:
+    title: Deployment options
+    description: 'The "Full" option creates both OCI and K8s resources.'
+    type: enum
+    enum: # Dev Note - # Any change in following options must be refactored across schema.yaml
+    - "Full"
+    - "Only OCI Resources"
+    default: "Full"
+    required: true
+    visible:
+       not: 
+       - livelab_switch    
+
   # Option to enable/disable metric server installation during helm deployment
   opt_deploy_metric_server:
     type: boolean
-    title: Enable Metric Server Installation
-    description: Uncheck this if Metric Server is already installed in your cluster.
+    title: Enable Metric Server installation
+    description: Clear this check box if Metric Server is already installed in your cluster.
     default: true
     visible:
       and:
-        - enable_helm_module
-        - not: 
-          - livelab_switch
+      - eq:
+        - ${stack_deployment_option}
+        - "Full"
+      - not: 
+        - livelab_switch
 
   # Compartment for creating OCI Observability and Management resources
   oci_onm_compartment_ocid:
     type: oci:identity:compartment:id
     required: true
-    title: Select compartment for Logging Analytics, Management Agent, and Monitoring service resources
-    description: This compartment will be used for creating Dashboards, Log Groups, Entities, Management Agent Keys, Metrics Namespace etc. See https://github.com/oracle-quickstart/oci-kubernetes-monitoring for full list of resources.
+    title: Select compartment for Logging Analytics, Management Agent, and Monitoring service resources.
+    description: "This compartment will be used for creating dashboards, log groups, entities, Management Agent keys, metric namespaces, and related resources. For a full list of resources, see: https://github.com/oracle-quickstart/oci-kubernetes-monitoring"
     default: compartment_ocid
 
   # Option to create Logging Analytics
   opt_create_new_la_logGroup: # change this to create new log group
     type: boolean
-    title: Check if you want to create a new Log Group
+    title: Select this check box if you want to create a new log group.
```

Comment:
we are creating a new log-group for each cluster so this checkbox should be inverted to say "Use existing log group"